src/Components/getList.js

```diff
@@ -1,29 +1,57 @@
 import React from 'react';
 import { FirestoreCollection } from 'react-firestore';
 import { getLocalToken } from '../lib/token.js';
+import { useHistory } from 'react-router-dom';
 
-class GetList extends React.Component {
-  render() {
-    return (
-      <FirestoreCollection
-        path={`${getLocalToken()}/`}
-        render={({ isLoading, data }) => {
-          return isLoading ? (
-            <div>loading...</div>
-          ) : (
-            <div>
-              <h1>Groceries</h1>
-              <ul>
-                {data.map(item => (
-                  <li key={item.id}>{item.itemName}</li>
-                ))}
-              </ul>
-            </div>
-          );
-        }}
-      />
-    );
+function EmptyList() {
+  let history = useHistory();
+
+  function handleClick() {
+    history.push('/add-item');
   }
+
+  return (
+    <div className="bg-white shadow pa2">
+      <h1 className="b f1">Welcome to your shopping list!</h1>
+      <p>
+        To get started add an item to your shopping list by clicking "add item
+        below"
+      </p>
+      <button onClick={handleClick} className="bg-green ph2 pv1 white f2 b">
+        Add an item
+      </button>
```

Comment:
No need to change, but, every time we redirect an user from routeA to routeB we should use an anchor instead of a button. 👍 This doesn't mean that you can't use buttons for this kind of things, but `Buttons` are great for forms, and actions and anchors for redirecting users. 🌱

Reply:
One rule of thumb I use is:
* If it has an href or redirects the user with no other logic, it's an `<a>` anchor tag
* If clicking the thing needs to execute some logic at any point, it's a `<button>`

Stylistically the two can be styled however they need to be styled to look, but semantically the two elements are good to keep distinct based on behavior.